controllers/whoToFollow/whoToFollowV2.js

```diff
@@ -54,12 +54,12 @@ module.exports = async (req, res) => {
                             a.is_anonymous = false
                             AND a.is_banned = false
                             AND a.blocked_by_admin = false
-                            AND a.karma_score > 30
+                            AND a.karma_score > 45
                             AND a.user_id != :admin_user_id
                             AND b.topic_id = :topic_id
                           ORDER BY 
-                            a.last_active_at DESC,
-                            CASE WHEN a.profile_pic_path != '%default-profile-picture%' THEN 0 ELSE 1 END,
+                            CASE WHEN a.last_active_at > now() - interval '7' day THEN 0 ELSE 1 END,
+                            CASE WHEN a.profile_pic_path NOT LIKE '%default-profile-picture%' THEN 0 ELSE 1 END,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the karma score threshold from 30 to 45 and the addition of a condition to prioritize users who have been active within the last 7 days are logical improvements for refining the user recommendation feature. However, it's important to note that these changes could potentially reduce the number of users being recommended, especially if there aren't many users with a karma score above 45 or those who have been active recently.

```diff
- AND a.karma_score > 30
+ AND a.karma_score > 45
- a.last_active_at DESC,
+ CASE WHEN a.last_active_at > now() - interval '7' day THEN 0 ELSE 1 END,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->